src/Shape/Shape.js

```diff
@@ -74,6 +75,7 @@ export default class Shape extends BaseClass {
     this._name = "Shape";
     this._opacity = constant(1);
     this._pointerEvents = constant("visiblePainted");
+    this._role = "presentation";
```

Comment:
make the default value use the constant function: `this._role = constant("presentation");`